.idea/.name

```diff
@@ -1 +1 @@
-LNMIIT
\ No newline at end of file
+LNMIIT_Android_App
```

Comment:
have you saw your commit message. "dding vectors in navigation drawer"? Please edit it.